h5p-bildetema/package.json

```diff
@@ -46,6 +47,7 @@
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
     "@vitejs/plugin-react": "^1.3.2",
+    "esbuild": "^0.14.43",
```

Comment:
What is this needed for?

Reply:
I think I installed it when I got the following message when trying to run Storybook:
```
The package "esbuild-windows-64" could not be found, and is needed by esbuild.

If you are installing esbuild with npm, make sure that you don't specify the
"--no-optional" flag. The "optionalDependencies" package.json feature is used
by esbuild to install the correct binary executable for your current platform.
The package "esbuild-windows-64" could not be found, and is needed by esbuild.
```
Then I found this topic and it was suggested to install esbuild - https://github.com/evanw/esbuild/issues/1646#issuecomment-1069353130